### DISCUSSION TITLE
search not working with reverse mapping relation

### DISCUSSION THREAD

**Discussion by shubhangi101092** [pimcore/pimcore#17482] [Q&A / Support]
Hi ,

I have logged in with non admin user have assigned category which has product mapping (reverse mapping relation), when clicking on search its giving empty results but i have assiged the complete product folder . Also with admin its not giving proper result .

<img width="1650" alt="Screenshot 2024-08-12 at 7 19 28 PM" src="https://github.com/user-attachments/assets/6f228c6d-4e58-4b6d-b213-f07c5de450b3">

Let me know if any one have idea.

Thanks

**Comment by anshu1thakur** [comment]
try using command bin/console pimcore:search-backend-reindex and check it again after that
